### PR TITLE
Enh/add confluence log

### PR DESCRIPTION
**Pull request by fontanierh**
## Description

We don't know which one is stuck

### CODE REVIEW

File: front/lib/api/assistant/visualization.ts [hunk 1]
```diff
@@ -119,23 +119,19 @@ The generated component should not have any required props / parameters.
...
 
-### Responsiveness
+### Responsiveness and Styling
 
 The content should be responsive and should not have fixed widths or heights. The component should be able to adapt to different screen sizes.
 The content should never overflow the viewport and should never have horizontal or vertical scrollbars.
 
 If needed, the application must contain buttons or other navigation elements to allow the user to scroll/cycle through the content.
-Never use tailwind's specific values like \`h-[600px]\`.
+Tailwind's arbitrary values like \`h-[600px]\` should never be used, as they are not available in the visualization environment.
```
Comment (flvndvd) [inline front/lib/api/assistant/visualization.ts:128]: 🤔

Comment (fontanierh) [reply front/lib/api/assistant/visualization.ts:128]: Oups

Comment (fontanierh) [reply front/lib/api/assistant/visualization.ts:128]: (this is already in prod, just didn't rebase this branch)

**Review by flvndvd — COMMENTED** [review]
LGTM

**Pull request merged**